RELEASE/scripts/sl_ascend/sl_tcrs.ash

```diff
@@ -186,6 +186,11 @@ boolean sl_knapsackAutoEat(boolean simulate)
 
 	if(fullness_left() == 0) return false;
 
+	if (item_amount($item[van key]) > 0)
+	{
+		use(min(5, item_amount($item[van key])), $item[van key]);

```

Comment:
Why not just `use(item_amount($item[van key]), $item[van key]);`? We'll ultimately end up using all of the van keys over the course of multiple evaluations anyway.